### ISSUE TITLE
history of OCaml

### ISSUE THREAD

**Issue by avsm** [ocaml/ood#50]
Not sure if this should be in ood or in the frontend, but there is a good history of OCaml in the v1 site:

https://caml.inria.fr/about/history.en.html

**Comment by patricoferris** [comment]
Yep probably in ood is fine, it is also on v2 https://github.com/ocaml/ocaml.org/blob/master/site/learn/history.md
